modules/statics/src/coins.ts

```diff
@@ -1006,6 +1006,26 @@ export const coins = CoinMap.fromCoins([
     BaseUnit.TCOREUM,
     COSMOS_SIDECHAIN_FEATURES
   ),
+  account(
+    '9fa0f191-4eed-4030-864a-d14bbd98c8af',
+    'rune-native',
+    'Rune',
+    Networks.main.rune,
+    8,
+    UnderlyingAsset.RUNE,
+    BaseUnit.RUNE,
```

Comment:
i think the base unit is called `tor`
https://docs.thorchain.org/how-it-works/thorchain-name-service#fees

Reply:
base unit is mentioned as tor in documents, but throchain apis always return the amount in `denom: rune`. Confirmed it with foundation team of thorchain